tests/__init__.py

```diff
@@ -1,14 +1,17 @@
-# run tests: pytest -sv --cov-report term-missing --cov=workflow-array-ephys -p no:warnings
+# dependencies: pip install pytest pytest-cov
+# run all tests: pytest -sv --cov-report term-missing --cov=workflow-array-ephys -p no:warnings tests/
+# run one test, debug: pytest [above options] --pdb tests/tests_name.py -k function_name
 
 import os
 import pytest
 import pandas as pd
 import pathlib
-import datajoint as dj
 import numpy as np
+import datajoint as dj
 
 import workflow_array_ephys
 from workflow_array_ephys.paths import get_ephys_root_data_dir
+import element_data_loader.utils
```

Comment:
```suggestion
from element_interface.utils import find_root_directory, find_full_path
```
With the recent package rename let's update from `element_data_loader` to `element_interface`.  And to keep consistent with the `elements`, let's import the function from the module.  Please update these scripts to incorporate these changes.